modules/service/main.tf

```diff
@@ -522,7 +522,7 @@ resource "aws_iam_role_policy_attachment" "service" {
 module "container_definition" {
   source = "../container-definition"
 
-  for_each = { for k, v in var.container_definitions : k => v if local.create_task_definition }
+  for_each = { for k, v in var.container_definitions : k => v if local.create_task_definition && try(v.create, true)}
```

Comment:
There is already a way to disable it by setting `var.create_task_definition` to `false` - https://github.com/terraform-aws-modules/terraform-aws-ecs/blob/master/modules/service/main.tf#L587

Reply:
I added an option to disable container definition, not task definition. For example, I have 3 container definitions, my appplication, datadog agent and firelens, but in DEV environment I disable Datadog agent container.

Reply:
@bryantbiggs WDYT?

Reply:
I guess its fine - just another bad practice brought by using named workspaces